### DISCUSSION TITLE
Organization name squatting

### DISCUSSION THREAD

**Discussion by IiTossiI** [community/community#65449] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So recently me and my friends created a project called AntiUtopia and today I wanted to claim the organization with such name but it was taken. When I checked it, it had 0 people, 0 repos and it was absoloutely empty. I tried to create a ticket but didnt find the choice for that anywhere. Is it possible to somehow contact github to free the organization name?

**Comment by alan-deronyan** [comment]
I have the same problem... is it possible to have an inactive org deleted or to transfer ownership?

**Comment by ptitjes** [comment]
Same question!

**Comment by mzzmjd** [comment]
Raise a support request with GitHub. 
Include details on why you think that it is organization squatting.
